vars/mavenCanaryRelease.groovy

```diff
@@ -52,7 +52,14 @@ def call(body) {
         bayesianScanner(body)
     }
 
-    sonarQubeScanner(body)
+    //use sonarqube waitforqualitygate stage in project jenkinsfile instead
+    //sonarQubeScanner(body)
```

Comment:
ok that change will go in the project jenkinsfile. I'll revisit the jenkinsfile in MovieManager, and make appropriate change as a demo.